homeassistant/components/ring/const.py

```diff
@@ -18,6 +18,7 @@
     Platform.BINARY_SENSOR,
     Platform.BUTTON,
     Platform.CAMERA,
+    Platform.EVENT,
```

Comment:
New platforms should get their own PR

Reply:
Done in https://github.com/home-assistant/core/pull/125506 which will need to rebased after this is merged